Tools/TestWebKitAPI/glib/WebKitGLib/wpe/WebViewTestWPE.cpp

```diff
@@ -76,13 +95,56 @@ static unsigned testMouseButtonToWPELegacy(WebViewTest::MouseButton button)
     RELEASE_ASSERT_NOT_REACHED();
 }
 
+#if ENABLE(WPE_PLATFORM)
+static WPEModifiers testModifiersToWPE(const OptionSet<WebViewTest::Modifiers>& modifiers)
+{
+    unsigned wpeModifiers = 0;
+    if (modifiers.contains(WebViewTest::Modifiers::Control))
+        wpeModifiers |= WPE_MODIFIER_KEYBOARD_CONTROL;
+    if (modifiers.contains(WebViewTest::Modifiers::Shift))
+        wpeModifiers |= WPE_MODIFIER_KEYBOARD_SHIFT;
+    if (modifiers.contains(WebViewTest::Modifiers::Alt))
+        wpeModifiers |= WPE_MODIFIER_KEYBOARD_ALT;
+    if (modifiers.contains(WebViewTest::Modifiers::Meta))
+        wpeModifiers |= WPE_MODIFIER_KEYBOARD_META;
+    return static_cast<WPEModifiers>(wpeModifiers);
+}
+
+static unsigned testMouseButtonToWPE(WebViewTest::MouseButton button)
+{
+    switch (button) {
+    case WebViewTest::MouseButton::Primary:
+        return WPE_BUTTON_PRIMARY;
+    case WebViewTest::MouseButton::Middle:
+        return WPE_BUTTON_MIDDLE;
+    case WebViewTest::MouseButton::Secondary:
+        return WPE_BUTTON_SECONDARY;
+    }
+    RELEASE_ASSERT_NOT_REACHED();
```

Comment:
I always assumed you still need a return value here to please the compiler? return 0?